db/migrate/20220913131104_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.text :title
+      t.text :text
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Please add foreign keys to create one-to-many relationships between posts and comments, likes. You can check out how to do this [here](https://guides.rubyonrails.org/active_record_migrations.html#references). For example, when relating posts with comments, run something similar to this in the terminal 
`rails generate migration AddPostRefToComments post:references`